### ISSUE TITLE
Add "id" as a configurable a11y option in order to pass snapshot tests

### ISSUE THREAD

**Issue by jonathanprizeout** [nolimits4web/swiper#5586]
### Clear and concise description of the problem

I love all of the accessibility features that come with Swiper but my snapshot tests are failing because of the dynamic id that is generated on the `swiper-wrapper` div when enabling `a11y`. The wrapper div is generated on the fly so I cannot control the markup in order to override the id manually. I would like to see an "id" as an option of `a11y`.

React Example:

```
const returnSlides = () => {
	return slides.map((slide) => (
		<SwiperSlide key={swiperSlideId}>
			...
		</SwiperSlide>
	));
};

<Swiper
	key={swiperId}
	grabCursor={true}
	slidesPerView={2.3}
	pagination={{ clickable: true }}
	navigation={true}
	scrollbar={true}
	mousewheel={false}
	spaceBetween={8}
	slidesOffsetBefore={16}
	slidesOffsetAfter={16}
	freeMode={true}
	freeModeMomentum={false}
	freeModeMomentumBounce={false}
	a11y={{ enabled: true }}
	breakpoints={{
		481: {
			slidesPerView: 'auto',
			spaceBetween: 16,
			slidesOffsetBefore: 32,
			slidesOffsetAfter: 32,
		},
	}}
>
	{returnSlides()}
</Swiper>
```

Screenshot of the DOM with dynamic wrapper and id

![image](https://user-images.githubusercontent.com/67278275/160426190-228c4629-ab9f-46f6-89e9-c9d04e22a327.png)

Screenshot of tests failing due to dynamic id

![image](https://user-images.githubusercontent.com/67278275/160429650-a313eae1-aff3-4714-bd77-7f6222c37df8.png)

### Suggested solution

Allow user to configure id to override dynamic id

```
a11y={
	{
		enabled: true,
		id: 'swiper-wrapper-id'
	}
}
```

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Would you like to open a PR for this feature?

- [x] I'm willing to open a PR

**Comment by perusoa** [comment]
I have this same issue, but currently I need to use v6.8.4, is there a workaround/solution I can implement in my version to do this by any chance?
